src/integrated/soil_canopy_model.jl

```diff
@@ -78,7 +78,7 @@ function soil_boundary_fluxes(
         t,
         model.parameters,
     )
-    net_energy_flux = @. p.soil_Rn + p.soil_lhf + p.soil_shf
+    net_energy_flux = @. -p.soil_LW_n - p.soil_SW_n + p.soil_lhf + p.soil_shf
```

Comment:
currently, LW_n and SW_n are positive if the soil absorbs energy. In the soil PDE, this must be a negative (downwards) flux, hence the minus signs here.

Reply:
net_energy_flux is G in this case?

Reply:
yes - Ill change the name in this PR too!